README.md

```diff
@@ -102,6 +102,41 @@ Repository::customApplicationData(fn () => [
 }
 ```
 
+## Heartbeats
+
+Configure `butler.health.heartbeat.url` and `butler.health.heartbeat.token` to enable.
+
+```php
+heartbeat('foo bar'); // POST http://heartbeat.localhost/foo-bar/1
+
+heartbeat('foo baz', 5); // POST http://heartbeat.localhost/foo-baz/5
+```
+
+### Fake
+
+Instead of faking the laravels [Http client](https://laravel.com/docs/master/http-client) in your tests you can fake heartbeats, see example below.
```

Comment:
```suggestion
Instead of faking the laravel [Http client](https://laravel.com/docs/master/http-client) in your tests you can fake heartbeats, see example below.
```